helm-chart/dash0-operator/templates/operator/gke-autopilot-allowlist-synchronizer.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if (and .Values.operator.gke.autopilot.enabled .Values.operator.gke.autopilot.deployAllowlistSynchronizer) }}
+apiVersion: auto.gke.io/v1
+kind: AllowlistSynchronizer
+metadata:
+  name: dash0-allowlist-synchronizer
+  annotations:
+    {{/*
+    We need to make sure that the AllowlistSynchronizer has been deployed (and is ready, i.e. has fetched the
```

Comment:
> and is ready, i.e. has fetched the referenced WorkloadAllowlist resources

is checking that part happening implicitly?

Reply:
Yes, as soon as the synchronizer is deployed, GKE fetches the referenced workload allowlists. The synchronizer becomes ready once the allow lists have been deployed. I'll add a comment to that extent.